marklogic-data-hub-central/ui/e2e/cypress/integration/curation/load/defaultIngestionCardView.spec.tsx

```diff
@@ -34,6 +34,11 @@ describe("Validate CRUD functionality from card view and run in a flow", () => {
     cy.resetTestUser();
     cy.waitForAsyncRequest();
   });
+  it("Verify Load tile is visible after navigation", () => {
+    toolbar.getLoadToolbarIcon().click();
+    cy.log("DHFPROD-8332: Every tile is getting rendered blank after navigation from Home");
+    loadPage.getContainerTitle().should("be.visible");
+  });
   it("Verify Cancel", () => {
     cy.waitUntil(() => toolbar.getLoadToolbarIcon()).click();
     cy.waitUntil(() => loadPage.stepName("ingestion-step").should("be.visible"));
```

Comment:
Can we delete the  "cy.waitUntil(()"? They are not necessary

Reply:
These lines weren't added originally for this fix, but removed them as per requested anyway.